easybuild/easyconfigs/g/GangSTR/GangSTR-2.4.4-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,32 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Pau Ruiz <pau.ruiz@hpcnow.com>, Alex Garcia
+# License::     GPL-v3.0
+# #
+
+easyblock = 'ConfigureMake'
+name = 'GangSTR'
+version = '2.4.4'
+
+homepage = 'https://github.com/gymreklab/GangSTR'
+description = """GangSTR is a tool for genome-wide profiling tandem repeats from short reads"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/gymreklab/GangSTR/releases/download/v2.4.4/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['47dda4e8eb94d4005cc9cc6673188cf564665d9f254f6cc88428a373cdef4b05']
+
```

Comment:
Please add `builddependencies = [('pkg-config', '0.29.2')]`, as this is used during configure.